src/core/ddsc/tests/qos.c

```diff
@@ -646,3 +649,214 @@ CU_Test(ddsc_qos, durability_service, .init=qos_init, .fini=qos_fini)
     CU_ASSERT_EQUAL_FATAL(p.max_samples_per_instance, g_pol_durability_service.max_samples_per_instance);
 }
 
+CU_Test(ddsc_qos, property, .init=qos_init, .fini=qos_fini)
+{
+    char * value = NULL;
+    char ** names = NULL;
+    uint32_t cnt = 0;
+    char * str;
+    const size_t len = 0x6400000; /* 100 mbyte */
```

Comment:
What's the point in allocation 100MB?

Reply:
I didn't realise that the value for `propagate` is currently always `0`, so the property name/values won't be serialized and sent over the wire and this 100 mbyte test does not make sense. I've removed this code from the test case.